apps/src/applab/Visualization.jsx

```diff
@@ -37,7 +48,7 @@ var Visualization = React.createClass({
     var appHeight = applabConstants.APP_HEIGHT - applabConstants.FOOTER_HEIGHT;
 
     return (
-      <ProtectedStatefulDiv>
+      <div>
```

Comment:
Makes me a little worried to remove the protection, but this should be okay because I'm still never changing the structure of the DOM - just the style attributes. Not sure if there's any good way to enforce that.